packages/helper-plugin/src/components/SidebarDropdown/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import Sidebar from '../Sidebar';
+
+const SidebarDropdown = ({
+  label,
+  children
+}) => {
+  const [opened, setOpened] = useState<boolean>(false);
+
+  return (
+    <Sidebar onClick={() => !opened && setOpened(true)}>
+      <Sidebar.ActionButton
+        label={label}
+        onClick={() => opened && setOpened(false)}
```

Comment:
Not really a fan of an `onClick` inside an element that has `onClick`. Maybe the sidebar element should have an `onOpen` prop instead?

Reply:
Done